go.mod

```diff
@@ -2,7 +2,12 @@ module github.com/rancher/rancher-operator
 
 go 1.15
 
-replace k8s.io/client-go => k8s.io/client-go v0.20.0
+replace (
+	github.com/rancher/rancher/pkg/apis => github.com/pennyscissors/rancher/pkg/apis v0.0.0-20210205233934-6d0193d42bf0
+	github.com/rancher/rancher/pkg/client => github.com/pennyscissors/rancher/pkg/client v0.0.0-20210205233934-6d0193d42bf0
+	github.com/rancher/rke => github.com/pennyscissors/rke v1.2.0-rc2.0.20210205224836-bc1782733189
```

Comment:
These are just placeholders to make sure I'm vendoring the right things. They will be removed once [this PR](https://github.com/rancher/rancher/pull/31138) is merged.